model/canvas.cc

```diff
@@ -503,7 +503,22 @@ namespace minsky
            });
 	}      
     itemVector.clear();  
-  }  
+  }
+  
+  void Canvas::showPlotsOnTab()
+  {
+    if (itemVector.empty())
```

Comment:
Why would you only do this if itemVector is empty? Wouldn't the user expect that all remaining plots be shown on the tab in this action?